scripts/multisim.sh

```diff
@@ -22,7 +22,7 @@ sim() {
   file="$tmpdir/gaia-simulation-seed-$seed-date-$(date -u +"%Y-%m-%dT%H:%M:%S+00:00").stdout"
   echo "Writing stdout to $file..."
 	go test ./cmd/gaia/app -run $testname -SimulationEnabled=true -SimulationNumBlocks=$blocks \
-    -SimulationVerbose=true -SimulationCommit=true -SimulationSeed=$seed -SimulationPeriod=5 -v -timeout 24h > $file
+    -SimulationVerbose=true -SimulationCommit=true -SimulationSeed=$seed -SimulationPeriod=100 -v -timeout 24h | tee $file
```

Comment:
We can't use a period of `100` with a `50`-block long sim (that doesn't test anything) - maybe make the period also a parameter.